fastrtps.repos

```diff
@@ -6,16 +6,16 @@ repositories:
     fastcdr:
         type: git
         url: https://github.com/eProsima/Fast-CDR.git
-        version: v1.0.24
+        version: 1.0.x
     fastrtps:
         type: git
         url: https://github.com/eProsima/Fast-DDS.git
         version: 2.6.x
     fastddsgen:
         type: git
         url: https://github.com/eProsima/Fast-DDS-Gen.git
-        version: v2.1.2
+        version: 2.1.x
     fastddsgen/thirdparty/idl-parser:
         type: git
         url: https://github.com/eProsima/IDL-Parser.git
-        version: v1.2.0
+        version: 1.2.x
```

Comment:
Im afraid that neither `fastddsgen` nor `idl-parser` have those branches. We can leave it with the fixed tags

Reply:
As we agreed, we would need to create those branches in the repos

Reply:
Note: We agreed to remove idl-parser from the .repos file, and create 2.1.x in Fast-DDS-Gen

Reply:
Exactly, I forgot updating my previous comment after making decision on what to do with `idl-parser`. Thanks !